src/lib/KeyStoreIndexeddb.js

```diff
@@ -181,10 +186,12 @@ class KeyStore {
         });
     }
 
-    close() {
-        if (!this._db) return;
-        this._db.close();
-        this._db = null;
+    /** @returns {Promise<void>} */
+    async close() {
+        if (!this._dbPromise) return;
+        const db = await this._dbPromise;
+        db.close();
+        this._dbPromise = null;
```

Comment:
DRY code...

Reply:
intended here because AccountStore is deprecated